azkaban-common/src/test/java/azkaban/metric/MetricManagerTest.java

```diff
@@ -72,6 +73,7 @@ public void managerEmitterMaintenanceTest() {
   /**
    * Test metric reporting methods, including InMemoryMetricEmitter methods
    */
+  @Ignore("TODO replace sleep with something better")
```

Comment:
There's a 2000ms sleep in this method..

Reply:
@kunkun-tang can you think of a way to make this test run faster?